docs/non-normative/database-test-cases/db-sql-test-cases.json

```diff
@@ -0,0 +1,315 @@
+        },
+        "expected": {
+            "db.query.text": [
+                "INSERT INTO shipping_details\n(order_id,\naddress)\nSELECT order_id,\naddress\nFROM   orders\nWHERE  order_id = ?"
+            ],
+            "db.query.summary": "INSERT shipping_details SELECT orders"
+        }
+    },
+    {
+        "name": "select_nested_query",
+        "input": {
+            "db.system.name": "other_sql",
+            "query": "SELECT order_date\nFROM   (SELECT *\nFROM   orders o\nJOIN customers c\nON o.customer_id = c.customer_id)"
+        },
+        "expected": {
+            "db.query.text": [
+                "SELECT order_date\nFROM   (SELECT *\nFROM   orders o\nJOIN customers c\nON o.customer_id = c.customer_id)"
+            ],
+            "db.query.summary": "SELECT SELECT orders customers"
+        }
+    },
+    {
+        "name": "select_nested_query_case_preserved",
+        "input": {
+            "db.system.name": "other_sql",
+            "query": "SELEcT order_date\nFROM   (sELECT *\nFROM   orders o\nJOIN customers c\nON o.customer_id = c.customer_id)"
+        },
+        "expected": {
+            "db.query.text": [
+                "SELEcT order_date\nFROM   (sELECT *\nFROM   orders o\nJOIN customers c\nON o.customer_id = c.customer_id)"
+            ],
+            "db.query.summary": "SELEcT sELECT orders customers"
+        }
+    },
+    {
+        "name": "case_preserved",
+        "input": {
+            "db.system.name": "other_sql",
+            "query": "SELEcT order_date\nFROM ORders"
+        },
+        "expected": {
+            "db.query.text": [
+                "SELEcT order_date\nFROM ORders"
+            ],
+            "db.query.summary": "SELEcT ORders"
+        }
+    },
+    {
+        "name": "cross_join",
+        "input": {
+            "db.system.name": "other_sql",
+            "query": "SELECT * FROM Orders o CROSS JOIN OrderDetails od"
+        },
+        "expected": {
+            "db.query.text": [
+                "SELECT * FROM Orders o CROSS JOIN OrderDetails od"
+            ],
+            "db.query.summary": "SELECT Orders OrderDetails"
+        }
+    },
+    {
+        "name": "cross_join_comma_separated_syntax",
+        "input": {
+            "db.system.name": "other_sql",
+            "query": "SELECT * FROM Orders o, OrderDetails od"
+        },
+        "expected": {
+            "db.query.text": [
+                "SELECT * FROM Orders o, OrderDetails od"
+            ],
+            "db.query.summary": "SELECT Orders OrderDetails"
+        }
+    },
+    {
+        "name": "create_table",
+        "input": {
+            "db.system.name": "other_sql",
+            "query": "CREATE  TABLE MyTable (\n    ID NOT NULL IDENTITY(1,1) PRIMARY KEY\n)"
+        },
+        "expected": {
+            "db.query.text": [
+                "CREATE  TABLE MyTable (\n    ID NOT NULL IDENTITY(?,?) PRIMARY KEY\n)"
+            ],
+            "db.query.summary": "CREATE  TABLE MyTable"
```

Comment:
while preserving case has benefits (some people prefer lower vs upper, and some systems might be case sensitive), do you think there's any downside to normalizing spaces?

Reply:
Right, forgot we discussed this. I normalized the whitespace in `db.query.summary` but left `db.query.text` alone. WDYT?